arm-relay/2016-07-01/relay.json

```diff
@@ -0,0 +1,2013 @@
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Description of a Namespace resource."
+    },
+    "NamespaceProperties": {
+      "properties": {
+        "Path": {
+          "type": "string",
+          "description": "The path of the relay."
+        },
+        "RelayType": {
+          "type": "string",
+          "description": "Relay Type.",
+          "enum": [
+            "NetTcp",
+            "Http"
+          ],
+          "x-ms-enum": {
+            "name": "Relaytype",
+            "modelAsString": false
+          }
+        },
+        "createdAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time the namespace was created."
+        },
+        "updatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The time the namespace was updated."
+        },
+        "ListenerCount": {
+          "format": "int32",
+          "type": "integer",
+          "description": "The number of listeners for this relay"
+        },
+        "RequiresClientAuthorization": {
+          "type": "boolean",
+          "description": "true if client authorization is needed for this relay; otherwise, false."
+        },
+        "AuthorizationRules": {
+          "$ref": "#/definitions/AuthorizationRules"
+        },
+        "RequiresTransportSecurity": {
+          "type": "boolean",
+          "description": "true if transport security is needed for this relay; otherwise, false."
+        },
+        "IsDynamic": {
+          "type": "boolean",
+          "description": "true if the relay is dynamic; otherwise, false."
+        },
+        "UserMetadata": {
+          "type": "string",
+          "description": "TheThe user metadata associated with this instance."
+        },
+        "CollectionName": {
+          "type": "string",
+          "description": "The name of the collection associated with the relay."
+        }
+      },
+      "description": "Properties of the Namespace."
+    },
+    "Sku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of this Sku",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "SkuName",
+            "modelAsString": true
+          }
+        },
+        "tier": {
+          "type": "string",
+          "description": "The tier of this particular SKU",
+          "enum": [
+            "Standard"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": true
+          }
+        }
+      },
+      "required": [
+        "name",
+        "tier"
+      ],
+      "description": "Sku of the Namespace."
+    },
+    "AuthorizationRules": {
+      "type": "object",
+      "properties": {
+        "RequiresEncryption": {
```

Comment:
should be camelCase. Please make sure that the casing is correct as per the wire format. Otherwise there will be serialization/deserialization issues.